pkg/services/publicdashboards/database/database.go

```diff
@@ -267,3 +268,20 @@ func (d *PublicDashboardStoreImpl) Delete(ctx context.Context, orgId int64, uid
 
 	return affectedRows, err
 }
+
+func (d *PublicDashboardStoreImpl) GetByDashboardFolder(ctx context.Context, dashboard *dashboards.Dashboard) ([]*PublicDashboard, error) {
+	if dashboard == nil || !dashboard.IsFolder {
+		return []*PublicDashboard{}, nil
```

Comment:
then I don't need to worry about getting panics bc I forgot to handle the nil case. I just iterate over result, so an empty slice works better in this case

Reply:
I think range doesn't panic if you want to iterate over a `nil` slice because the `len` of it is 0. Also this case may be different because I'm used to using slices of values instead of slices of pointers. 
When you define below `var pubdashes []*PublicDashboard` that's `nil` so if there is no error and the find does not override the value is it possible you are returning `nil` at the end? Not sure tbh

Reply:
nice, can iterate over nil safely 👍